docs/UserGuide.md

```diff
@@ -266,6 +268,14 @@ Examples:
 * `freeSchedule ti/ 12:00`
 * `freeSchedule ti/ 14:00 da/2022-02-14`
 
+### Exporting a person's schedule: `export`
```

Comment:
Remember to update the command summary as well

Reply:
Alright, thanks